docker/localnode/scripts/step1_configure_init.sh

```diff
@@ -4,63 +4,62 @@
 MONIKER="sei-node-$NODE_ID"
-./build/seid init "$MONIKER" --chain-id sei
+seid init "$MONIKER" --chain-id sei
 
 # Copy configs
+ORACLE_CONFIG_FILE="build/node_$NODE_ID/price_feeder_config.toml"
 cp docker/localnode/config/app.toml ~/.sei/config/app.toml
 cp docker/localnode/config/config.toml ~/.sei/config/config.toml
-cp docker/localnode/config/price_feeder_config.toml ~/price_feeder_config.toml
+cp docker/localnode/config/price_feeder_config.toml "$ORACLE_CONFIG_FILE"
 
 # Set up persistent peers
-SEI_NODE_ID=$(./build/seid tendermint show-node-id)
+SEI_NODE_ID=$(seid tendermint show-node-id)
 NODE_IP=$(hostname -i | awk '{print $1}')
 echo "$SEI_NODE_ID@$NODE_IP:26656" >> build/generated/persistent_peers.txt
 
 # Create a new account
 ACCOUNT_NAME="node_admin"
 echo "Adding account $ACCOUNT_NAME"
-printf "12345678\n12345678\ny\n" | ./build/seid keys add "$ACCOUNT_NAME" >/dev/null 2>&1
+printf "12345678\n12345678\ny\n" | seid keys add "$ACCOUNT_NAME" >/dev/null 2>&1
 
 # Get genesis account info
-GENESIS_ACCOUNT_ADDRESS=$(printf "12345678\n" | ./build/seid keys show "$ACCOUNT_NAME" -a)
+GENESIS_ACCOUNT_ADDRESS=$(printf "12345678\n" | seid keys show "$ACCOUNT_NAME" -a)
 echo "$GENESIS_ACCOUNT_ADDRESS" >> build/generated/genesis_accounts.txt
 
 # Add funds to genesis account
-./build/seid add-genesis-account "$GENESIS_ACCOUNT_ADDRESS" 10000000usei
+seid add-genesis-account "$GENESIS_ACCOUNT_ADDRESS" 10000000usei
 
 # Create gentx
-printf "12345678\n" | ./build/seid gentx "$ACCOUNT_NAME" 10000000usei --chain-id sei
+printf "12345678\n" | seid gentx "$ACCOUNT_NAME" 10000000usei --chain-id sei
 cp ~/.sei/config/gentx/* build/generated/gentx/
 
 # Creating some testing accounts
 echo "Creating $NUM_ACCOUNTS accounts"
-python3 loadtest/scripts/populate_genesis_accounts.py $NUM_ACCOUNTS loc >/dev/null 2>&1
+python3 loadtest/scripts/populate_genesis_accounts.py "$NUM_ACCOUNTS" loc >/dev/null 2>&1
 echo "Finished $NUM_ACCOUNTS accounts creation"
 
-# Set node0 seivaloper info
-NODE0_SEIVALOPER_INFO=$(printf "12345678\n" | ./build/seid keys show "$ACCOUNT_NAME" --bech=val -a)
-PRIV_KEY=$(printf "12345678\n12345678\n" | ./build/seid keys export "$ACCOUNT_NAME")
-echo "$PRIV_KEY" >> build/generated/exported_keys/"$NODE0_SEIVALOPER_INFO".txt
+# Set node seivaloper info
+SEIVALOPER_INFO=$(printf "12345678\n" | seid keys show "$ACCOUNT_NAME" --bech=val -a)
+PRIV_KEY=$(printf "12345678\n12345678\n" | seid keys export "$ACCOUNT_NAME")
+echo "$PRIV_KEY" >> build/generated/exported_keys/"$SEIVALOPER_INFO".txt
 
 # Update price_feeder_config.toml with address info
-sed -i'' -e 's/address = "sei"/address = \"'$GENESIS_ACCOUNT_ADDRESS'\"/g' ~/price_feeder_config.toml
-sed -i'' -e 's/validator = "seivaloper"/validator = \"'$NODE0_SEIVALOPER_INFO'\"/g' ~/price_feeder_config.toml
+sed -i.bak -e "s|^address *=.*|address = \"$GENESIS_ACCOUNT_ADDRESS\"|" $ORACLE_CONFIG_FILE
+sed -i.bak -e "s|^validator *=.*|validator = \"$SEIVALOPER_INFO\"|" $ORACLE_CONFIG_FILE
```

Comment:
do we also want to be setting up a feeder account? if we're not needing to perform any TXs from the validator accounts then its not necessary otherwise it might conflict with the seq num for price feeder

Reply:
Good call, is there any special process for setting a feeder account? Maybe we can change to that in a different PR